flang/lib/Lower/CallInterface.cpp

```diff
@@ -28,10 +28,33 @@ bool Fortran::lower::CallerInterface::hasAlternateReturns() const {
   return procRef.hasAlternateReturns();
 }
 
+static std::string getBindCName(const Fortran::semantics::Symbol &symbol) {
+  // bind(C) name expression, if any, must be a scalar constant of default
+  // character
+  // type as per R808.
```

Comment:
nit: weird formatting.

Reply:
Corrected thanks! I generally leave the formatting to clang-format, seems like this one get away with clang-format too :)